testnets/cascadiatestnet/chain.json

```diff
@@ -0,0 +1,150 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "cascadiatestnet",
+  "status": "live",
+  "network_type": "testnet",
+  "website": "https://www.cascadia.foundation/",
+  "pretty_name": "Cascadia",
+  "chain_id": "cascadia_6102-1",
+  "bech32_prefix": "cascadia",
+  "daemon_name": "cascadiad",
+  "node_home": "$HOME/.cascadiad",
+  "key_algos": [
+    "ethsecp256k1"
+  ],
+  "slip44": 60,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "aCC",
+        "fixed_min_gas_price": 0,
+        "low_gas_price": 0.05,
+        "average_gas_price": 0.05,
+        "high_gas_price": 0.1
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "tCC"
```

Comment:
The correction you've implemented have made it much more clear which is the base unit of this token. The staking token must also refer to a base denom, so this should be aCC instead?

Reply:
yeah, aCC, fixed